pkgs/development/tools/rust/cargo-llvm-cov/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, rustPlatform, fetchCrate, fetchFromGitHub, libllvm, makeBinaryWrapper }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-llvm-cov";
+  version = "0.4.14";
+
+  src = fetchCrate {
+    inherit pname version;
+    sha256 = "sha256-DY5eBSx/PSmKaG7I6scDEbyZQ5hknA/pfl0KjTNqZlo=";
+  };
+
+  postPatch = ''
+    rm tests/test.rs
+  '';
+
+  nativeBuildInputs = [ makeBinaryWrapper ];
+  buildInputs = [ libllvm ];
+
+  LLVM_COV = "${rustPlatform.rust.rustc.llvmPackages.llvm}/bin/llvm-cov";
+  LLVM_PROFDATA = "${rustPlatform.rust.rustc.llvmPackages.llvm}/bin/llvm-profdata";
```

Comment:
What are these required for, if the resulting binary is wrapped? Does the build require this?

Reply:
They would be required for the tests (which are disabled at the moment).